packages/config/config/devices/0x0403/plha10000.json

```diff
@@ -0,0 +1,100 @@
+{
+	"manufacturer": "ABUS Security-Center GmbH & Co. KG",
+	"manufacturerId": "0x0403",
+	"label": "PLHA10000",
+	"description": "Z-Wave 230V relay module",
+	"devices": [
+		{
+			"productType": "0x0003",
+			"productId": "0x0005",
+			"zwaveAllianceId": 3679
+		}
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"associations": {
+		"1": {
+			"label": "Lifeline",
+			"maxNodes": 5,
+			"isLifeline": true
+		},
+		"2": {
+			"label": "key 1",
+			"maxNodes": 5
+		},
+		"3": {
+			"label": "key 2",
+			"maxNodes": 5
+		}
+	},
+	"paramInformation": [
+		{
+			"#": "1",
+			"label": "Key 1",
+			"description": "0: momentary button, 1: switch (toggle)",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 0,
+			"unsigned": true
+		},
+		{
+			"#": "2",
+			"label": "Key 2",
+			"description": "0: momentary button, 1: switch (toggle)",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 0,
+			"unsigned": true
+		},
+		{
+			"#": "3",
+			"label": "Relay 1 Auto Off Time",
+			"description": "0: no auto off",
+			"valueSize": 4,
```

Comment:
Missing the unit, the description should be an option instead.

Reply:
could be an option have only 1 entry and also set an integer?